client/modules/IDE/components/CollectionList/CollectionListRow.jsx

```diff
@@ -79,7 +80,7 @@ class CollectionListRowBase extends React.Component {
 
   handleCollectionDelete = () => {
     this.closeAll();
-    if (window.confirm(`Are you sure you want to delete "${this.props.collection.name}"?`)) {
+    if (window.confirm(this.props.t('CollectionListRow.ConfirmDelete', { key_label: this.props.collection.name }))) {
```

Comment:
We use this same delete confirmation text in many places in the app. I think this is a good candidate for moving to `Common`.

If any component needs to have a variation on that, they can still just use their own translation key.

I actually [did this already in PR #1549](https://github.com/processing/p5.js-web-editor/pull/1549/files#diff-36da9b5b481aaff253f0d1b49d52961fR203) but it hasn't been merged yet so why not change them here, and I'll update my PR to match.

Reply:
Yes, now that #1549 is merged I changed it.